File: 0344-reverse-string/0344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        char temp;
        int size = s.size();
        for (int i = 0; i < size / 2; i++) {
            temp = s[i];
            s[i] = s[size-1-i];
            s[size-1-i] = temp;
        }
    }
};

File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        sz: int = len(s)-1
        temp: str = ""
        for i in range(len(s)//2):
            temp = s[i]
            s[i] = s[sz-i]
            s[sz-i] = temp